integration-tests/plugin-tests/src/test/resources/test-enclave/build.gradle

```diff
@@ -24,6 +24,7 @@ java {
 conclave {
     productID = 1
     revocationLevel = 0
+    runtime = findProperty("runtimeType").toString()  // This is provided by AbstractPluginTaskTest
```

Comment:
Should we lowercase here? I think we do that in another test

Reply:
Doesn't matter either way, since the parsing of this enum ignores case. And in this build script we're not using the property elsewhere in the script, unlike in integration-tests/build.gradle